packages/components/input/src/input.tsx

```diff
@@ -89,7 +94,7 @@ const Input = forwardRef<"input", InputProps>((props, ref) => {
 
     return (
       <>
-        <div {...getInputWrapperProps()}>
+        <div className="bg-blue-900" {...getInputWrapperProps()}>
```

Comment:
what is the reason of adding `bg-blue-900` here?

Reply:
I was testing something and was trying to understand the components.
Looks like I forgot to remove these when I made a commit.
Will change them A.S.A.P.